src/components/Table/TitledCell.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {Cell} from './styles';
+
+const withTitle = (Component: React.ComponentType<any>) => ({
+  children,
+  ...rest
+}: any) => (
+  <Component title={React.Children.toArray(children).join('')} {...rest}>
+    {children}
+  </Component>
+);
+
+export const TitledCell = withTitle(Cell);
```

Comment:
good catch but this is done wrong

you need:
`export cont withTitle`
`export default withTitle(Cell)`
at least

then you don't omit `withTitle` which is also usable in many cases